timed/permissions.py

```diff
@@ -134,3 +134,11 @@ class IsNotTransferred(BasePermission):
 
     def has_object_permission(self, request, view, obj):
         return not obj.transferred
+
+
+class IsBilledAndVerfied(BasePermission):
```

Comment:
This permission is only ever used in negated form, so building the negated version would make more sense.
There is also a typo: `IsNotBilledAndVerified`

Reply:
Thanks!